src/client.ts

```diff
@@ -1,39 +1,189 @@
-import axios from 'axios';
+/// <reference path="../types/modules.d.ts" />
+
+import * as axios from 'axios';
 import http from 'http';
 import https from 'https';
-import Faye from 'faye';
+import * as Faye from 'faye';
 import jwtDecode from 'jwt-decode';
 
 import Personalization from './personalization';
-import BatchOperations from './batch_operations';
 import Collections from './collections';
-import StreamFeed from './feed';
 import StreamFileStore from './files';
 import StreamImageStore from './images';
 import StreamReaction from './reaction';
 import StreamUser from './user';
 import createRedirectUrl from './redirect_url';
 import signing from './signing';
-import errors from './errors';
+import * as errors from './errors';
 import utils from './utils';
-import pkg from '../../package.json';
+import BatchOperations, { FollowRelation, UnfollowRelation } from './batch_operations';
+import StreamFeed, {
+  UpdateActivity,
+  Activity,
+  EnrichOptions,
+  PersonalizationFeedAPIResponse,
+  GetActivitiesAPIResponse,
+  GetFeedOptions,
+} from './feed';
+
+// no import since typescript json loader shifts the final output structure
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const pkg = require('../package.json');
```

Comment:
This seems like a step back. How about awaiting import ?

Reply:
unfortunately not gonna work, I fixed this in chat SDK by using rollup replace plugin, not ideal by will see if we can do the same with webpack

Reply:
actually this is not an option because webpack is only used for browser dist version, we can live with circumventing typescript for this one

Reply:
can we add a TODO to see this later?